internal/components/base/reconcile.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2021 Cisco Systems, Inc. and/or its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package base
+
+import (
+	"emperror.dev/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/banzaicloud/istio-operator/v2/api/v1alpha1"
+	"github.com/banzaicloud/istio-operator/v2/internal/static/gen/charts/base"
+	"github.com/banzaicloud/operator-tools/pkg/helm"
+	"github.com/banzaicloud/operator-tools/pkg/helm/templatereconciler"
+	"github.com/banzaicloud/operator-tools/pkg/types"
+)
+
+const (
+	ComponentName = "base"
+	ChartName     = "base"
+	ReleaseName   = "istio-operator-base"
```

Comment:
I think these all could be unexported variables at this point.
```suggestion
	componentName = "base"
	chartName     = "base"
	releaseName   = "istio-operator-base"
```